src/directives/ec-tooltip/ec-tooltip.story.ts

```diff
@@ -11,56 +12,59 @@ export default {
       options: defaultPlacements,
       control: { type: 'select' },
     },
-    trigger: {
-      options: ['hover', 'click'],
+    triggers: {
+      options: [TooltipTrigger.HOVER, TooltipTrigger.CLICK],
       control: { type: 'select' },
     },
     popperClass: {
       options: [
-        '',
-        'ec-tooltip--level-1',
-        'ec-tooltip--level-2',
-        'ec-tooltip--level-3',
-        'ec-tooltip--bg-bright',
-        'ec-tooltip--bg-success',
-        'ec-tooltip--bg-error',
+        TooltipPopperClass.EC_TOOLTIP_BG_BRIGHT,
+        TooltipPopperClass.EC_TOOLTIP_BG_ERROR,
+        TooltipPopperClass.EC_TOOLTIP_BG_SUCCESS,
+        TooltipPopperClass.EC_TOOLTIP_LEVEL_1,
+        TooltipPopperClass.EC_TOOLTIP_LEVEL_2,
+        TooltipPopperClass.EC_TOOLTIP_LEVEL_3,
       ],
       control: { type: 'select' },
     },
   },
 };
 
-const Template = ({ trigger, ...args }) => ({
+const Template = ({ triggers, ...args }: { triggers: TooltipTrigger}) => ({
   directives: { EcTooltip },
   inheritAttrs: false,
   setup() {
-    return { args, trigger };
+    return { args, triggers };
   },
   template: `
     <div class="tw-m-80">
       <div
         class="tw-p-12 tw-rounded tw-text-gray-8 tw-my-auto tw-mx-20 tw-bg-additional-18"
-        v-ec-tooltip="{ ...args, triggers: [trigger] }">Hover over this element to see the tooltip.</div>
+        v-ec-tooltip="{ ...args, triggers: [triggers] }">Hover over this element to see the tooltip.</div>
     </div>
   `,
 });
 
 export const basic = Template.bind({});
 
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
 basic.args = {
   content: 'Test tooltip',
   shown: true,
-  placement: 'top',
+  placement: TooltipPlacement.TOP,
   delay: 100,
   popperClass: '',
-  trigger: 'hover',
+  triggers: TooltipTrigger.HOVER,
```

Comment:
Why it is plural now?

Reply:
it just matches the object key name used in the ec-tooltip, otherwise we'll have trigger inside the storybook, but we'll need to use triggers for the directive